modules/ROOT/pages/starlight.adoc

```diff
@@ -0,0 +1,25 @@
+= Starlight for Luna Streaming
+
+Starlight for Luna Streaming allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform with various connectors for Apache Pulsar™.
+
+== Starlight for RabbitMQ
+
+Starlight for RabbitMQ™ combines the industry-standard AMQP 0.9.1 (RabbitMQ) API with the cloud-native and horizontally scalable Apache Pulsar™ streaming platform, providing a powerful way to modernize your RabbitMQ infrastructure, improve performance, and reduce costs.
+
+Starlight for RabbitMQ™ is open source and is included in DataStax’s Luna Streaming Enterprise support for Pulsar. 
```

Comment:
Does this apply to Starlight for JMS? What about when Starlight for Kafka is added? If all of the Starlight connectors are included in the support plan, let's move this to the first section and make it more generic. Let's also link to the support policy or another Luna Streaming reference similar. Probably https://www.datastax.com/legal/luna-streaming-supplemental-support-policy, but confirm with @chrislatimer.

Reply:
@mendonk Let's also link to the support policy or another Luna Streaming reference similar. Probably https://www.datastax.com/legal/luna-streaming-supplemental-support-policy, but confirm with @chrislatimer.